Makefile

```diff
@@ -34,7 +34,9 @@ install:
 	# copy static files verbatim
 	/bin/cp -a static/* $(DESTDIR)
 	mkdir -p $(DESTDIR)/install-data
-	/bin/cp -r $(CRAFT_STAGE)/local-debs $(DESTDIR)/install-data/local-debs
+	if [ -d $(CRAFT_STAGE)/local-debs ]; then \
+		/bin/cp -r $(CRAFT_STAGE)/local-debs $(DESTDIR)/install-data/local-debs; \
+	fi
```

Comment:
There shouldn't be anymore local debs, maybe we can remove this now?

Reply:
A followup maybe?

Reply:
Yep that is fine